pkg/scheduler/scheduler.go

```diff
@@ -156,6 +156,17 @@ func (s *PubSubScheduler) Schedule(campaign db.Campaign) error {
 }
 
 func (s *PubSubScheduler) publishTask(ctx context.Context, task *db.Task) error {
+
+	//Check if task.CampaignID belongs to a cancelled/halted Campaign. If so skip it.
```

Comment:
Comments should begin with `// ` (with a space).

Reply:
Possible linter rule for future check-ins?